### DISCUSSION TITLE
We are getting some errors on keda-operator-metrics-server

### DISCUSSION THREAD

**Discussion by deepakpandeysymplr** [kedacore/keda#3703] [General]
We are getting this error, can anyone help me to resolve

**E0928 12:00:40.114839       1 azure_servicebus_scaler.go:227] keda_metrics_adapter/azure_servicebus_scaler "msg"="error getting service bus entity length" "error"="Get \"http://169.254.169.254/metadata/identity/oauth2/token?api-version=2018-02-01&resource=https%3A%2F%2Fservicebus.azure.net%2F\": context deadline exceeded (Client.Timeout exceeded while awaiting headers)" "name"="XYZ" "namespace"="XYZ" "type"="ScaledObject"**

**Comment by JorTurFer** [comment]
Could you share your ScaledObject? This seems that the configuration is incorrect. Do you have any firewall blocking the traffic?
